cross/ffmpeg-2.2.1/Makefile

```diff
@@ -0,0 +1,36 @@
+PKG_NAME = ffmpeg
+PKG_VERS = 2.2.1
+PKG_EXT = tar.gz
+PKG_DIST_NAME = $(PKG_NAME)-$(PKG_VERS).$(PKG_EXT)
+PKG_DIST_SITE = http://www.ffmpeg.org/releases/
+PKG_DIR = $(PKG_NAME)-$(PKG_VERS)
+
+DEPENDS = cross/x264 cross/lame
+
+HOMEPAGE = http://www.ffmpeg.org/
+COMMENT  = FFmpeg is a complete, cross-platform solution to record, convert and stream audio and video. It includes libavcodec - the leading audio/video codec library
+LICENSE  =
+
+REQUIRED_DSM = 5.1
+
+# Define SYNCTHING_ARCH as per SyncThing standards
+ifeq ($(findstring $(ARCH),88f6281),$(ARCH))
+MYARCH = armv5
```

Comment:
This caught my eye (didn't check the rest of the PR too closely yet). 
I could be mistaken, but my earlier investigation showed that ffmpeg doesn't care at all about which arm arch you're trying to compile, just that it contains `arm`. The reason for the `FFMPEG_ARCH` patches was that the word `armada` is not recognized further on in the compile process, or something like that (I forgot already...)
If that's correct, you could just lump all the arm-based arches together and do `FFMPEG_ARCH=arm`.